swimlane/core/adapters/app.py

```diff
@@ -24,7 +24,11 @@ def get(self, key, value):
         Raises:
             TypeError: No or multiple keyword arguments provided
             ValueError: No matching app found on server
+            ValueError: The lookup value is empty or None
         """
+        if not value:
+            raise ValueError('No {} provided'.format(key))
```

Comment:
Would it be any clearer to use something like `'The value provided for the key "{0}" cannot be empty or None'` here?